ios/RNSScreen.mm

```diff
@@ -951,6 +956,18 @@ - (void)viewDidDisappear:(BOOL)animated
 #else
   [self traverseForScrollView:self.screenView];
 #endif
+
+  // See: https://github.com/software-mansion/react-native-screens/pull/1797
+  // In brownfield appliactions there are some flows in which React Native mechanisms
```

Comment:
typo `appliactions` -> `applications`

Reply:
Hey, I've moved fix of this issue to react native's repository: https://github.com/facebook/react-native/pull/38617